pkg/ring/ring.go

```diff
@@ -475,3 +478,13 @@ func (r *Ring) GetInstanceState(instanceID string) (IngesterState, error) {
 
 	return instance.GetState(), nil
 }
+
+// Includes returns whether the ring includes the instance with the provided instanceID.
```

Comment:
Was this meant to say "HasInstance returns" instead of "Includes returns"?

Reply:
Uh ops, thanks for noticing it! I initially named the function `Includes` and then, after renaming, I forgot to update the comment 🤦